cmake/mcuboot.cmake

```diff
@@ -111,6 +115,24 @@ if(CONFIG_BOOTLOADER_MCUBOOT)
     --pad-header
     )
 
+  if(CONFIG_ZIGBEE)
+    set(zb_sign_cmd
+      ${PYTHON_EXECUTABLE}
+      ${NRF_DIR}/scripts/bootloader/zb_ota_sign.py
+      generate
+      --application ${PROJECT_BINARY_DIR}/app_signed.hex
+      --application-version-string ${CONFIG_ZIGBEE_APP_VERSION}
+      --hw-version ${CONFIG_ZIGBEE_HARDWARE_VERSION}
+      --zigbee-ota-hw-version ${CONFIG_ZIGBEE_OTA_HW_VERSION}
+      --zigbee-ota-fw-version ${CONFIG_ZIGBEE_OTA_FW_VERSION}
+      --zigbee-ota-min-hw-version ${CONFIG_ZIGBEE_OTA_MIN_HW_VERSION}
```

Comment:
@mariuszpos Will it be possible to have `--zigbee-ota-min-hw-version` and `--zigbee-ota-max-hw-version` as options, so these attributes may not present in the Zigbee OTA header?

Reply:
Rationale: For jenkins tests we need to have it as options, so different sets of paramaters can be tested (no min/max hw version, min hw-version only etx.)

Reply:
Done